lib/js/components/Survey-Questions/SurveyQuestionOpenEnded/SurveyQuestionOpenEnded.vue

```diff
@@ -46,8 +46,11 @@
 @import '../../../../styles/settings/buttons';
 @import '../../../../styles/settings/spacings';
 @import '../../../../styles/settings/typography';
+@import '../../../../styles/settings/colors/tokens';
 
 .surveyQuestionOpenEnded {
+	color: $color-default-text;
```

Comment:
this color should be here by default. Why do we need to override it?

Reply:
Please verify this problem in all places

Reply:
@kvas-damian I know the reason. Rules from `design-system.scss` are not applied in SB. Rules from `storybook.scss` are applied. It's weird because theses files are imported in the same file `preview.js`:
```
import 'design-system/lib/styles/design-system.scss';
import 'design-system/lib/styles/storybook.scss';
```

Reply:
yes, I mentioned that in https://github.com/bethinkpl/design-system/pull/115#discussion_r848372348 ;)